src/main/java/net/imagej/legacy/convert/ResultsTableColumnWrapper.java

```diff
@@ -0,0 +1,345 @@
+		}
+		return true;
+	}
+
+	@Override
+	public boolean addAll(final Collection<? extends Object> c) {
+		boolean b = false;
+		for (final Object d : c) {
+			b = b || add(d);
+		}
+		return b;
+	}
+
+	@Override
+	public boolean addAll(final int index, final Collection<? extends Object> c) {
+		throw new UnsupportedOperationException("addAll(int, Collection)");
+	}
+
+	@Override
+	public boolean removeAll(final Collection<?> c) {
+		throw new UnsupportedOperationException("removeAll(Collection)");
+	}
+
+	@Override
+	public boolean retainAll(final Collection<?> c) {
+		throw new UnsupportedOperationException("retainAll(Collection)");
+	}
+
+	@Override
+	public void clear() {
+		// Determine if empty cells are NaN or 0
+		double fill = 0;
+		try {
+			final Field f = ij.measure.ResultsTable.class.getDeclaredField(
+				"NaNEmptyCells");
+			f.setAccessible(true);
+			final boolean nan = (boolean) f.get(table);
+			fill = nan ? Double.NaN : 0;
+		}
+		catch (final Exception exc) {
+			// Keep as zero
+		}
+
+		for (int i = 0; i < table.size(); i++) {
+			// set strings to "", this must be done first. Since setting the string
+			// sets the corresponding double value position to NaN
+			table.setValue(col, i, "");
+			table.setValue(col, i, fill);
+		}
+	}
+
+	@Override
+	public Object get(final int index) {
+		if (checkString(index)) return table.getStringValue(col, index);
+		return table.getValueAsDouble(col, index);
+	}
+
+	/**
+	 * Sets the value at the given index. If {@code element} is a {@code Number}
+	 * it will be stored as {@code double}.
+	 */
+	@Override
+	public Object set(final int index, final Object element) {
+		// Store the previous value
+		Object prev = null;
+		if (checkString(index)) prev = table.getStringValue(col, index);
+		else prev = table.getValueAsDouble(col, index);
+
+		// Set the new value
+		if (element instanceof String) table.setValue(col, index, (String) element);
+		else if (element instanceof Number) table.setValue(col, index,
+			((Number) element).doubleValue());
+		else throw new IllegalArgumentException("element must be String or Double");
+
+		return prev;
+	}
+
+	@Override
+	public void add(final int index, final Object element) {
+		throw new UnsupportedOperationException("add(int, Object)");
+	}
+
+	@Override
+	public Double remove(final int index) {
+		throw new UnsupportedOperationException("remove(int)");
+	}
+
+	@Override
+	public int indexOf(final Object o) {
+		if (o instanceof String) {
+			final String s = (String) o;
+			for (int i = 0; i < table.size(); i++) {
+				if (s.equals(table.getStringValue(col, i))) return i;
+			}
+		}
+		else if (o instanceof Number) {
+			final double d = ((Number) o).doubleValue();
+			for (int i = 0; i < table.size(); i++) {
+				if (d == table.getValueAsDouble(col, i)) return i;
+			}
+		}
+		// if not String, Double, or in column return -1
+		return -1;
+	}
+
+	@Override
+	public int lastIndexOf(final Object o) {
+		if (o instanceof String) {
```

Comment:
No worries! I consolidated `lastIndexOf` to be `findInRange(o, IntStream.range(0, table.size()).map(i -> table.size() - i - 1));` and got rid of the `reverse` method. So it's actually good it was the other way, because it gave me a pattern to follow since I don't know much about streams. :smile_cat: